drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -337,7 +338,21 @@ static void start_ble_rf(void)
 	LL_RCC_SetRTCClockSource(LL_RCC_RTC_CLKSOURCE_LSE);
 
 	/* Switch OFF LSI */
-	LL_RCC_LSI1_Disable();
```

Comment:
Was this a bugfix?

Reply:
Got your point. Yes, actually LSI_1 is used by RTC, but LSI2 is used by RF. Thus there is no sense in touching LSI1 in BLE. Moved to separate commit